app.json

```diff
@@ -9,7 +9,10 @@
                 "postdeploy": "python manage.py setup_dev"
             },
             "env": {
-                "SELF_CAPTURE": true
+                "SELF_CAPTURE": true,
+                "SECRET_KEY": {
+                    "generator": "secret"
+                }
```

Comment:
There's also `env` at the very bottom, so I suppose that was useless and can be removed?

Reply:
Nope, that's for when you deploy the actual instance. The review app was pulling the secret key from there, until self_capture was added.